lib/region.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+'use strict'
+
+const apis = {
+  AU: 'https://auth-api.blinkm.io',
+  US: 'https://us-auth-api.blinkm.io'
+}
+
+class RegionError extends Error {
+  constructor(message) {
+    super(message)
+    this.name = 'RegionError'
+  }
+}
+
+module.exports = (
+  regionCode /* : ?string */ = 'AU',
+  apiOrigin /* : mixed */
+) => {
+  if (!!regionCode && !Object.keys(apis).includes(regionCode)) {
+    throw new RegionError(
```

Comment:
I think a `TypeError` is sufficient here. Is there somewhere else in our code base we are catching `RegionError`s?

Reply:
Nope I'll use ```TypeError``` I think ```RegionError``` was being used earlier but the design changed